analysis-module/src/analysis.py

```diff
@@ -111,7 +112,11 @@ def compute_propagation_times(msgs_dict):
 
     for msg_id, msg_data in pbar:
         pbar.set_description('Computing propagation time of message %s' % msg_id)
-        msg_propagation_times.append(round(max(msg_data['latencies']) / 1000000))
+        if msg_data['latencies']:
+            msg_propagation_times.append(max(msg_data['latencies']) / 1000000)
+            # msg_propagation_times.append(round(max(msg_data['latencies']) / 1000000))
```

Comment:
Just to know if maybe I missunderstood someting, which was the reason of doing a round here before?